core/src/main/scala/Kubectl.scala

```diff
@@ -64,14 +64,14 @@ final class Kubectl(mode: KubernetesMode) {
   }
 
   def getDeployment(namespace: NamespaceName, stackName: StackName): IO[DeploymentStatus] =
-    exec(List("kubectl", "get", "deployment", stackName.toString, "-n", namespace.root.asString, "-o", "json"), emptyStdin)
+    exec(List("kubectl", "get", "deployment", stackName.toString, "--context", namespace.root.asString, "-o", "json"), emptyStdin)
       .flatMap(_.output)
       .flatMap { stdout =>
         IO.fromEither(Parse.decodeEither[DeploymentStatus](stdout.mkString("\n")).leftMap(kubectlJsonError))
       }
 
   def getCronJob(namespace: NamespaceName, stackName: StackName): IO[JobStatus] =
-    exec(List("kubectl", "get", "job", "-l", s"stackName=${stackName.toString}", "-n", namespace.root.asString, "-o", "json"), emptyStdin)
+    exec(List("kubectl", "get", "job", "-l", s"stackName=${stackName.toString}", "--context", namespace.root.asString, "-o", "json"), emptyStdin)
```

Comment:
actually this is *very* breaking... this assumes that the kubeconfig files people are using are structured in a particular manner, correct? This would break my setup today.